clinica/iotools/converter_utils.py

```diff
@@ -96,7 +96,10 @@ def print_longitudinal_analysis(
             for subject in subjects_avail:
                 subj_tsv_path = path.join(bids_dir, subject, f"{subject}_sessions.tsv")
                 subj_df = pd.read_csv(subj_tsv_path, sep="\t").set_index("session_id")
-                if ses in subj_df.index.values:
+                if (
+                    ses in subj_df.index.values
+                    and "diagnosis" in subj_df.columns.values
+                ):
```

Comment:
That has nothing to do with this PR right?

Reply:
Looks like your first commit should have been in a separate PR maybe?

Reply:
I realized this problem existed when I wrote the documentation, this is why I added it here.